src/main/utils/mavenUtils.ts

```diff
@@ -0,0 +1,369 @@
+    }
+
+    public static getRawDependenciesList(pathToPomXml: string, treesManager: TreesManager): string[] {
+        try {
+            let result: string[] = exec
+                .execSync('mvn dependency:tree', { cwd: pathToPomXml })
+                .toString()
+                .split('\n')
+                .filter(line => line.includes('[INFO]'))
+                .join('\n')
+                .split('[INFO] \n')
+                .map(text => text.replace(/\[INFO\]/g, ''))
+                .filter(text => text.includes('maven-dependency-plugin'));
+            result[result.length - 1] = result[result.length - 1].split('------')[0];
+            return result;
+        } catch (error) {
+            treesManager.logManager.logError(error, false);
+            treesManager.logManager.logMessage(
+                `Could not get dependencies tree from pom.xml (path: ${path.dirname(pathToPomXml)}).
+            'Possible cause: The project needs to be installed by maven. Install it by running "mvn clean install" from ${path.dirname(
+                pathToPomXml
+            )}.`,
+                'ERR'
+            );
+            return [];
+        }
+    }
+
+    /**
+     * @param workspaceFolders - Base workspace folders
+     * @param progress         - Progress bar
+     * @param componentsToScan - Set of maven components to populate during the tree building. We'll use this set later on, while scanning the packages with Xray.
+     * @param scanCacheManager - Scan cache manager
+     * @param root           - The base tree node
+     * @param quickScan        - True to allow using the scan cache
+     */
+    public static async createDependenciesTrees(
+        workspaceFolders: vscode.WorkspaceFolder[],
+        progress: vscode.Progress<{ message?: string; increment?: number }>,
+        componentsToScan: Collections.Set<ComponentDetails>,
+        treesManager: TreesManager,
+        root: DependenciesTreeNode,
+        quickScan: boolean
+    ): Promise<MavenTreeNode[]> {
+        let pomXmls: Collections.Set<vscode.Uri> = await MavenUtils.locatePomXmls(workspaceFolders, progress);
+        if (pomXmls.isEmpty()) {
+            treesManager.logManager.logMessage('No pom.xml files found in workspaces.', 'DEBUG');
+            return [];
+        }
+        treesManager.logManager.logMessage('pom.xml files to scan: [' + pomXmls.toString() + ']', 'DEBUG');
+        if (!MavenUtils.verifyMavenInstalled()) {
+            vscode.window.showErrorMessage('Could not scan Maven project dependencies, because "mvn" is not in the PATH.');
+            return [];
+        }
+        let mavenTreeNodes: MavenTreeNode[] = [];
+        progress.report({ message: 'Analyzing pom.xml files' });
+        let prototyeTree: PomTree[] = this.buildPrototypePomTree(pomXmls.toArray(), treesManager);
+        for (let ProjectTree of prototyeTree) {
+            progress.report({ message: 'Analyzing pom.xml files' });
```

Comment:
You already have this log in line 145. I suggest to remove the log in 145 and log here the path to the pom file that you're analyzing.